MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -102,35 +143,48 @@ class CarPlayMapViewController: UIViewController {
         resetCamera(animated: false, altitude: CarPlayMapViewController.defaultAltitude)
     }
     
-    override func viewWillDisappear(_ animated: Bool) {
+    override public func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
         styleObservation = nil
     }
     
-    public func zoomInButton() -> CPMapButton {
-        let zoomInButton = CPMapButton { [weak self] (button) in
-            guard let strongSelf = self else {
-                return
+    /**
+     Creates a new pan map button for the CarPlay map view controller.
+     
+     - parameter mapTemplate: The map template available to the pan map button for display.
+     */
+    @discardableResult public func createPanMapButton(for mapTemplate: CPMapTemplate) -> CPMapButton {
+        let panButton = CPMapButton { _ in
+            if !mapTemplate.isPanningInterfaceVisible {
+                mapTemplate.showPanningInterface(animated: true)
             }
-            strongSelf.mapView.setZoomLevel(strongSelf.mapView.zoomLevel + 1, animated: true)
         }
+        
         let bundle = Bundle.mapboxNavigation
-        zoomInButton.image = UIImage(named: "carplay_plus", in: bundle, compatibleWith: traitCollection)
-        return zoomInButton
+        panButton.image = UIImage(named: "carplay_pan", in: bundle, compatibleWith: traitCollection)
+        self.panMapButton = panButton
+        
+        return panButton
     }
     
-    public func zoomOutButton() -> CPMapButton {
-        let zoomInOut = CPMapButton { [weak self] (button) in
-            guard let strongSelf = self else {
-                return
-            }
-            strongSelf.mapView.setZoomLevel(strongSelf.mapView.zoomLevel - 1, animated: true)
+    /**
+     Creates a new close button to dismiss the visible panning buttons on the map.
+     
+     - parameter mapTemplate: The map template available to the pan map button for display.
+     */
+    @discardableResult public func createDismissPanningButton(for mapTemplate: CPMapTemplate) -> CPMapButton {
```

Comment:
“Create” is a noise word for a method name, like “get”. On the other hand, it would be confusing to start the method name with a verb, like “dismiss”. So how about `mapButton(forDismissingPanningInterfaceIn:)` or `panningInterfaceDismissalButton(for:)`?